apps/admin/frontend/src/app_write_in_flows.test.tsx

```diff
@@ -172,7 +172,7 @@ test('manual write-in data end-to-end test', async () => {
     .closest('tr')!;
   expect(
     within(precinct1SummaryRow).getByTestId('numBallots').textContent
-  ).toEqual('18');
+  ).toEqual('11');
```

Comment:
I couldn't make sense of this test well enough to understand if these changes are reasonable or indicate a bug

Reply:
I think this points to an underlying bug where our method of determining the total number of ballots in a precinct based on the total number of ballots in each contest is flawed. 

Basically, there should be 18 ballots in this precinct because we have 11 ballots from the mammal party and 7 ballots from the fish party. With the way we're calculating the total number, however, it sees the shared race (the now non-partisan races) and thinks they're the same ballot style, so it takes the maximum, 11. 

@jonahkagan this could take some thinking to get right so I'm not sure how you want to handle. Feel free to create a ticket and tag @adghayes and @carolinemodic, who may have more context. 

https://github.com/votingworks/vxsuite/blob/d43d42816b6a4d82d8491b9aa60448d5b70c93fc/apps/admin/frontend/src/utils/external_tallies.ts#L68-L122

Reply:
Ok, since this is a known bug now, I'm gonna not block this PR on it